dal/daltest/db_test.go

```diff
@@ -1,18 +1,19 @@
-package dal
+package daltest
 
 import (
 	"testing"
 
-	"gotest.tools/assert"
+	"github.com/gonative-cc/relayer/dal"
+	"gotest.tools/v3/assert"
 )
 
```

Comment:
I didn't mean to move tests. Only the helper functions

Reply:
https://github.com/gonative-cc/relayer/pull/35#discussion_r1862292962

Reply:
then move the tests to `dal/db_test.go` and type `dal_test` as the package name in test files.